new/LIT6201Hymn.xml

```diff
@@ -0,0 +1,60 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6201Hymn" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Hymn to the Miracles of ʾEwosṭātewos</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AD"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A set of hymns varying in length dedicated to the miracles of ʾEwosṭātewos.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AD" when="2020-10-14">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+          <div type="bibliography"/>
+          <listRelation>
+            <relation name="ecrm:P129_is_about" active="LIT6201Hymn" passive="PRS3934Ewostate"/>
```

Comment:
relations and/or abstract should list the hymns that form part of the collection 
                   
 ```
<relation name="saws:contains" active="LIT6201Hymn" passive="....">
                        <desc>The collection may contain "...."</desc>
```

Reply:
Note that in accordance with https://github.com/BetaMasaheft/Documentation/issues/1518
the MS descriptions should be updated with both IDs (with nesting introduced)
e.g. EMIP 764
```
<msItem....>
<title ref="LIT6201Hymn"/>
   <msItem... .1>
  <title ref="LIT6200Hymn"/>
  </msItem>
  <msItem... .2>
    </msItem>
  <msItem... .3>
    </msItem>
  <msItem... .4>
    </msItem>
</msItem>

```

Reply:
Precisely. In the case of BL Or 702, though, this would introduce three levels of nesting since LIT6201 would be `ms_i1.2.1`, not sure that I've seen such deep nesting elsewhere on BM?

Reply:
I was going to update the relations in a new branch after this one was merged, moving the files out from `new` and in the process updating the Chaîne records as well.

Reply:
we have even deeper levels of nesting :)